rpcserver.go

```diff
@@ -1939,6 +1939,7 @@ func (r *rpcServer) ListChannels(ctx context.Context,
 			NumUpdates:            localCommit.CommitHeight,
 			PendingHtlcs:          make([]*lnrpc.HTLC, len(localCommit.Htlcs)),
 			CsvDelay:              uint32(dbChannel.LocalChanCfg.CsvDelay),
+			Initiator:             dbChannel.IsInitiator,
```

Comment:
are there any other changes we need to support this? or is this already in the database, just not exposed to the rpc?

Reply:
@treygriffith from what I can tell, this was just not exposed in RPC (I think we had actually noticed this in december during the 0.5.1 upgrade)

PR is here: https://github.com/lightningnetwork/lnd/pull/2473

I dont think PendingChannels (or other channel calls supports this).